hack/enable-machine-autoscaling.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+set -eou pipefail
```

Comment:
```suggestion
set -eoux pipefail
```
Will allow for easier debugging if script has problems running in CI

Reply:
Yeah, good catch - added in the last force-push.